res/res_respoke_session.c

```diff
@@ -819,6 +820,28 @@ int respoke_session_offer(struct respoke_session *session)
 
 int respoke_session_answer(struct respoke_session *session)
 {
+	if (session->channel)
+	{
+		ast_verbose("\n\n<-- Respoke Session Answer: %s -->\n\n", ast_channel_name(session->channel));
```

Comment:
Generally, you shouldn't use ast_verbose directly, nor format the verbose statement in a special way. If you'd like this to be a VERBOSE level message, I'd use ast_verb(4:

```
ast_verb(4, "Respoke Session Answer for '%s'\n", ast_channel_name(session->channel));
```